features/aanschrijfwijze/hoffelijkheidstitel.feature

```diff
@@ -0,0 +1,331 @@
+      | type                | RaadpleegMetBurgerservicenummer  |
+      | burgerservicenummer | 000000231                        |
+      | fields              | adressering.aanschrijfwijze.naam |
+      Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
+      | naam                          | waarde                    |
+      | aanschrijfwijze.naam          | <naam in aanschrijfwijze> |
+      | aanschrijfwijze.aanspreekvorm | <aanspreekvorm>           |
+
+      Voorbeelden:
+      | titel partner | naam in aanschrijfwijze    | aanspreekvorm            |
+      | B             | A. barones van den Aedel   | De hoogwelgeboren vrouwe |
+      | G             | A. gravin van den Aedel    | De hooggeboren vrouwe    |
+      | H             | A. hertogin van den Aedel  | De hoogwelgeboren vrouwe |
+      | M             | A. markiezin van den Aedel | De hoogwelgeboren vrouwe |
+      | P             | A. prinses van den Aedel   | De hoogheid              |
+
+    Abstract Scenario: Vrouw met partner met adellijke titel "<titel partner>" kan geen hoffelijkheidstitel gebruiken
+      Gegeven de persoon met burgerservicenummer '000000231' heeft de volgende gegevens
+      | naam                        | waarde |
+      | geslachtsaanduiding (04.10) | V      |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                           | waarde     |
+      | voornamen (02.10)              | Anne-Fleur |
+      | voorvoegsel (02.30)            | de         |
+      | geslachtsnaam (02.40)          | Boer       |
+      | aanduiding naamgebruik (61.10) | P          |
+      En de persoon heeft een 'partner' met de volgende 'naam' gegevens
+      | naam                                 | waarde          |
+      | adellijke titel of predicaat (02.20) | <titel partner> |
+      | voorvoegsel (02.30)                  | van den         |
+      | geslachtsnaam (02.40)                | Aedel           |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                           |
+      | type                | RaadpleegMetBurgerservicenummer  |
+      | burgerservicenummer | 000000231                        |
+      | fields              | adressering.aanschrijfwijze.naam |
+      Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
+      | naam                          | waarde                    |
+      | aanschrijfwijze.naam          | <naam in aanschrijfwijze> |
+
+      Voorbeelden:
+      | titel partner | naam in aanschrijfwijze    |
+      | BS            | A. van den Aedel           |
+      | GI            | A. van den Aedel           |
+      | HI            | A. van den Aedel           |
+      | JH            | A. van den Aedel           |
+      | JV            | A. van den Aedel           |
+      | MI            | A. van den Aedel           |
+      | PS            | A. van den Aedel           |
```

Comment:
idem...

Reply:
waarom twijfel je specifiek bij hertogin en prinses?

Reply:
Is een uiting van verbazing, geen voorstel voor verbetering oid. Het is wat het is.